packages/salesforcedx-vscode-core/src/commands/describeMetadata.ts

```diff
@@ -11,10 +11,6 @@ import * as path from 'node:path';
 import { SfCommandletExecutor } from './util';
 
 class DescribeMetadataExecutor extends SfCommandletExecutor<string> {
-  public constructor() {
```

Comment:
linter is pretty smart.  no-useless-constructor looked and said, "oh, the parent class doesn't have a constructor, so this isn't actually doing anything"